.github/disabled-workflows/build_and_test_v2.yml

```diff
@@ -0,0 +1,436 @@
+name: Build and test v2
```

Comment:
Do we want to preserve these (I assume now redundant) workflows, or can we remove them?

Reply:
We want to keep these, but they are disabled for now until we start active development on the `develop2` branch.

Reply:
I misunderstood their use! Got it now, thanks.